index.js

```diff
@@ -66,6 +85,6 @@ function basic404 (req, reply) {
 }
 
 module.exports = fp(zipkinPlugin, {
-  fastify: '>=0.40.0',
+  fastify: '>= 2',
```

Comment:
```suggestion
  fastify: '^2.0.0',
```

So that it fails on Fastify v3.